modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -587,7 +587,6 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
                 int samplesPerStripe = std::max((batchSize + nstripes - 1)/nstripes, 1);
                 r.start *= samplesPerStripe;
                 r.end *= samplesPerStripe;
-                nstripes *= samplesPerStripe;
```

Comment:
@dkurt , could you please confirm that `nstripes` should not be modified in this block? Or should it be `nstripes_` instead?

Reply:
@mshabunin, with `nstripes_ *= samplesPerStripe` the following tests are failed. So I think it shouldn't be modified and we can remove this line

```
[  FAILED  ] Test_TensorFlow_layers.conv/0, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_CPU)
[  FAILED  ] Test_TensorFlow_nets.MobileNet_SSD/0, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_CPU)
[  FAILED  ] Test_TensorFlow_nets.Inception_v2_Faster_RCNN/0, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_CPU)
[  FAILED  ] Reproducibility_MobileNet_SSD.Accuracy/0, where GetParam() = DNN_TARGET_CPU
[  FAILED  ] DNNTestNetwork.MobileNet_SSD_Caffe/0, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL)
[  FAILED  ] DNNTestNetwork.MobileNet_SSD_Caffe/1, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16)
[  FAILED  ] DNNTestNetwork.MobileNet_SSD_v1_TensorFlow/0, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL)
[  FAILED  ] DNNTestNetwork.MobileNet_SSD_v1_TensorFlow/1, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16)
[  FAILED  ] DNNTestNetwork.MobileNet_SSD_v2_TensorFlow/0, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL)
[  FAILED  ] DNNTestNetwork.MobileNet_SSD_v2_TensorFlow/1, where GetParam() = (DNN_BACKEND_OPENCV, DNN_TARGET_OPENCL_FP16)
```